app/helpers/reactjs_helper.rb

```diff
@@ -6,23 +6,16 @@ def mount_react_component(name, selector, data = [])
   end
 
   def webpacked_plugins_js_for(*plugin_names)
-    tags = create_tags(plugin_names)
-    return nil if tags.empty?
-    tags.inject(&:concat).html_safe
+    js_tags_for(select_requested_plugins(plugin_names)).join.html_safe
   end
 
-  def create_tags(plugin_names)
-    js_tags_for select_requested_plugins(all_webpacked_plugins.map(&:id), plugin_names)
-  end
-
-  def select_requested_plugins(webpacked_plugin_ids, plugin_names)
-    plugin_names.select { |plugin_name| webpacked_plugin_ids.include?(plugin_name) }
-  end
-
-  def all_webpacked_plugins
-    Foreman::Plugin.registered_plugins.values.select do |plugin|
-      File.exist? File.join(plugin.path, 'webpack/index.js')
+  def select_requested_plugins(plugin_names)
+    available_plugins = Foreman::Plugin.with_webpack.map(&:id)
+    missing_plugins = plugin_names - available_plugins
+    if missing_plugins.any?
+      logger.error { "Failed to include webpack assets for plugins: #{missing_plugins}" }
```

Comment:
Perhaps in development mode this should actually be an exception so you can't miss it?